pkg/util/ovs/ovs.go

```diff
@@ -106,7 +107,7 @@ type ovsExec struct {
 }
 
 // New returns a new ovs.Interface
-func New(execer exec.Interface, bridge string, minVersion string) (Interface, error) {
+func New(execer exec.Interface, bridge string, minVersion string) (*ovsExec, error) {
```

Comment:
Hm... isn't that bad style? Maybe you could just have TestBundle() cast it from Interface to *ovsExec.

Alternatively, after the next commit, TestBundle() doesn't really do anything beyond what TestTransactionSuccess() and TestTransactionFailure() do, so you could just drop TestBundle() (and maybe add a TestEmptyTransaction() in the next commit if you want to explicitly test that case too.)